wasm/src/re2/re2/bitstate.cc

```diff
@@ -108,9 +108,11 @@ void BitState::Push(int id, const char* p) {
   if (njob_ >= job_.size()) {
     GrowStack();
     if (njob_ >= job_.size()) {
+#if 0
```

Comment:
nit: These kind of changes make me nervous when it comes time to update the library. Would it be possible to swap in our own util/logging.h and override the macro implementation with our own no-op thing?

Reply:
I thought about that. Unfortunately once the C++ streams are used, they bring a whole another set of additional C++ standard library dependencies and hence opted for this approach.

Reply:
Right, that makes sense. I guess I had envisioned us providing a no-op "std::ostream" class with stream operator overloads for the types used by re2 we compile with instead of the real deal. I'm assuming that i'm over simplifying things though..

Anyway, we can always revisit it if/when this becomes an issue. Definitely not something we need to figure out right now.

Reply:
Let me try this again, now that I have all the other dependencies figured. Perhaps it is better than I assume -- as I never tried the approach by implementing it, only thought about it.

Reply:
Tried it again: including ostream triggers an additional set of dependencies to libc around locale, so probably not worth to pursue this route further at this point.